### ISSUE TITLE
Bug: in_axes calculation in `klax.fit`.

### ISSUE THREAD

**Issue by Drenderer** [Drenderer/klax#19]
Currently the in_axes calculation in fit contains a bug: When the model input is a tuple then x=(x1, x2) and in_axes=(0,0) however, jax.vmap in the loss_fn interprets a tuple of in_axes as multiple arguments for the vmapped function.
Solution: Wrap in_axes into a single tuple.

**Comment by Drenderer** [comment]
The bug is fixed with commit a4f99bf3c95f92904301ec2293a30ed8b78ce4d6.
The default loss functions now pass in_axes as a single tuple into vmap.
